static/app/utils/issueTypeConfig/cronConfig.tsx

```diff
@@ -39,7 +39,7 @@ const cronConfig: IssueCategoryConfigMapping = {
       events: {enabled: true},
       openPeriods: {enabled: false},
       uptimeChecks: {enabled: false},
-      cronChecks: {enabled: false},
+      cronChecks: {enabled: true},
```

Comment:
can we rename this one to `cronCheckIns` too

Reply:
Quite a few files had to be updated, but done as well!